src/app/account-app/account-app.component.html

```diff
@@ -17,8 +31,9 @@
         [fixedInViewport]="mobileQuery.matches"
         fixedTopGap="0"
         id="sideMenu"
-        style="width: 330px"
-    >
+		 >
+        <h3>Settings Menu</h3>
```

Comment:
This looks a bit out of place imo (and we don't have a similar "menu title" on any other page).

If we want to make sure that the user knows that they're in the settings, we could try making the header toolbar Settings icon stand out somehow when it's the page we're on (like showing the text not only on hover)?

Reply:
I was ambivalent about adding it, but reasoned as follows:

1. It is important that the user immediately understands that the left-pane menu is used to navigate Settings and not the rest of the app, since the items will increasingly mirror the main menu items. (The need to clearly set Settings apart from the rest of the app was one reason for suggesting an "overlay" effect when accessing Settings.)
2. The main content area needed top padding to add vertical space above the heading "Account Settings". Consequentially the left-pane menu looked oddly attached to the top header, and equally odd with a similar padding without a heading on its own.
3. None of the other areas of the app (Mail, Contacts, Calendar) have a *menu* per se; although they have left panes, these mainly contain items for navigating various data selections (aside from the individual Settings items which should in the future take the user to Account Settings).
4. Adding similar headings in the other areas of the app would certainly look misplaced, and I don't think we should do that.

In short: I agree that adding a Settings Menu heading breaks convention but that it might be warranted, unless a better solution can be found that take the above 4 concerns into account.

I also agree with the suggestion to indicate which main menu item has been selected, and presumably this requires some Angular logic in order to be robust? :)

Reply:
I think it's clear that you are in settings menu as you click the settings icon to get there. I'm not sure if there was a question here about whether there should be a settings icon in the main menu but if there was I think it is correct it is there as Settings is a part of the app just like Mail, Calendar etc. and it has a section of it's own just like the others.

I don't think there is much to get confused about here once you've clicked on the Settings icon once (with or without a heading) :)

Reply:
> I think it's clear that you are in settings menu as you click the settings icon to get there. 

Not necessarily, as you could also (in the near future) have clicked a button elsewhere in the app (e.g. Mail or Contacts), or followed a link to https://runbox.com/app/account, to get to Settings.

> I'm not sure if there was a question here about whether there should be a settings icon in the main menu but if there was I think it is correct it is there as Settings is a part of the app just like Mail, Calendar etc. and it has a section of it's own just like the others.

I submit that Settings is **not** a part of the app just like Mail, Calendar etc -- rather that it is conceptually a meta-part as it controls how the other areas function (similar to Control Panel in Windows or System Preference on macOS), which was part of the initial reason for gathering all settings in one place. :)

> I don't think there is much to get confused about here once you've clicked on the Settings icon once (with or without a heading) :)

I think there is some room for confusion and that we should strive towards increased clarity especially where there might be a perception of redundancy or overlap, which I think is the case here.

Compare the following screenshot of the current Settings menu, while hovering over the Mail main menu item:
<img width="619" alt="Runbox 7 Settings menu current" src="https://user-images.githubusercontent.com/1620058/118683899-9c59e500-b7b6-11eb-82c4-a502b4a9a222.png">

...with the proposed one:

<img width="684" alt="Runbox 7 Settings menu proposed" src="https://user-images.githubusercontent.com/1620058/118683965-ab409780-b7b6-11eb-8844-94a79a83d209.png">

In the first screenshot the main menu looks similar to the Settings menu, while in the second the difference is clearer.

Additionally, the purely design-related comments above are aesthetically relevant.

Reply:
Go with with your suggestion then. I have no strong feelings about this and just wanted to convey that I think once you've visited the section it will be clear how to use it. I was going to suggest that maybe there should be a different icon for mail settings on the left so that it isn't so similar to the one in the main menu but I don't think that is possible as we aren't using custom icons?

It just occurred to me that if you are not sure which section the option you are looking for is under an "expand all" option might be useful (different topic though entirely).